packages/clay-layout/src/Sheet.tsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+interface IContainerProps extends React.HTMLAttributes<HTMLElement> {
+	/**
+	 * Element or component to render for container
+	 */
+	containerElement?: React.JSXElementConstructor<{
+		className: string;
+		[key: string]: any;
+	}>;
+}
+
+const Header: React.FunctionComponent<IContainerProps> = ({
+	children,
+	className,
+	containerElement: ContainerElement = 'div',
+	...otherProps
+}) => {
+	return (
+		<ContainerElement
+			{...otherProps}
+			className={classNames(className, 'sheet-header')}
+		>
+			{children}
+		</ContainerElement>
+	);
+};
+
+Header.displayName = 'ClaySheetHeader';
+
+const Footer: React.FunctionComponent<IContainerProps> = ({
+	children,
+	className,
+	containerElement: ContainerElement = 'div',
+	...otherProps
+}) => {
+	return (
+		<ContainerElement
+			{...otherProps}
+			className={classNames(
+				className,
+				'sheet-footer',
+				'sheet-footer-btn-block-sm-down'
```

Comment:
Is this always needed? This actually feels oddly specific. I can't find any usage of this in DXP... but @pat270 can call me out on this 😉

Reply:
It was a specific request for https://projects.invisionapp.com/d/main#/projects/prototypes/12432242. It hasn't been deprecated. If we don't need it, I'm good with removing it if Lexicon approves.
<img width="111" alt="sheet-block-btn" src="https://user-images.githubusercontent.com/788266/82079719-277e3780-9698-11ea-8faa-f025b46cdd40.png">

Reply:
I meant if this class was really a default one, and always ncessary.

We might need to keep it, I’m just not sure if it’s a default... I’ll take a closer look at DXP next week, but we might just pass it through ‘className’ for now if in doubt...

Reply:
OH I see. I would say don't default to `sheet-footer sheet-footer-btn-block-sm-down`. It's very specific to that form.